Assets/Scripts/HUD Scripts/QuantityDisplayScript.cs

```diff
@@ -157,7 +157,12 @@ public void UpdateInfo(GameObject obj, GameObject targetInfo, int index = 0)
             if (PartIndexScript.CheckPartObtained(info))
             {
                 targetName.text = info.partID;
-                targetDesc.text = AbilityUtilities.GetAbilityNameByID(info.abilityID, null) + " " + info.tier;
```

Comment:
It should be possible to just get rid of the abilityID == 10 check and pass in info.secondaryData in both cases.